eventuate-common-jdbc/src/main/java/io/eventuate/common/jdbc/sqldialect/PostgresDialect.java

```diff
@@ -15,4 +17,18 @@ public boolean supports(String driver) {
   public int getOrder() {
     return Integer.MIN_VALUE;
   }
+
+  @Override
+  public String castToJson(String sqlPart) {
+    return sqlPart + "::json";
```

Comment:
This needs to be conditional on the column type being JSON.

Reply:
ok